Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -142,26 +146,23 @@ import BraintreeDataCollector
         apiClient.post("/v1/payment_methods/paypal_accounts", parameters: parameters) { body, response, error in
             if let error = error as? NSError {
                 if error.code == BTCoreConstants.networkConnectionLostCode {
-                    self.apiClient.sendAnalyticsEvent("ios.paypal.handle-browser-switch.network-connection.failure")
+                    self.apiClient.sendAnalyticsEvent(BTPayPalAnalytics.tokenizeNetworkConnectionFailed)
                 }
 
-                self.apiClient.sendAnalyticsEvent("ios.\(paymentType.stringValue).tokenize.failed")
+                self.apiClient.sendAnalyticsEvent(BTPayPalAnalytics.tokenizeFailed)
                 completion(nil, error)
                 return
             }
 
-            self.apiClient.sendAnalyticsEvent("ios.\(paymentType.stringValue).tokenize.failed")
-            
             guard let payPalAccount = body?["paypalAccounts"].asArray()?.first,
                   let tokenizedAccount = BTPayPalAccountNonce(json: payPalAccount) else {
+                self.apiClient.sendAnalyticsEvent(BTPayPalAnalytics.tokenizeFailed)
                 completion(nil, BTPayPalError.failedToCreateNonce)
                 return
             }
-
-            if tokenizedAccount.creditFinancing != nil {
-                self.apiClient.sendAnalyticsEvent("ios.\(paymentType.stringValue).credit.accepted")
-            }
-            
+          
+            self.apiClient.sendAnalyticsEvent(BTPayPalAnalytics.browserLoginSucceeded)
```

Comment:
This _feels_ like a weird place for this event. I wonder if it'd make more sense after the error and cancel at the top of this method. That way it reads as: there was a URL, it was valid, and it wasn't canceled so the browser login was successful. Thoughts?